apps/demo/src/app/checkout/components/forms/address-form/components/address-form/address-form.component.scss

```diff
@@ -45,11 +45,12 @@
     }
   }
 
-  & input {
+  input {
     width: 100%;
   }
 
-  & select {
+  select {
     margin: 0;
+    height: 53px;
```

Comment:
We shouldn't have to apply a fixed height here, what's the reasoning?

Reply:
I can't add the same padding values as input into select because native selects don't allow padding. It also doesn't inherit the height of the containers surrounding it if I use `<daff-form-field>`.